global.ini

```diff
@@ -28451,386 +28451,386 @@ PU_GENOUTLAW4_M_CP_Chat_ProcConvo_OutlawSet_FollowUp_Neg_IG_001_YeahIDont=Ja, ic
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Low_IG_001_LetsGetBack=Lass uns zurück zur Mission kommen, okay?
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Low_IG_002_ComeOnWeve=Komm schon, wir haben einen Job zu erledigen. Ich sage das nicht nochmal.
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Low_IG_003_LookLetsGet=Schau mal, lass uns das erledigen, okay? Ich sage es nicht nochmal.
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Low_IG_004_IfYouDont=Wenn du den Job nicht machst, bekommst du kein Geld. Verstanden?
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Low_IG_005_IveHiredYou=Ich habe dich eingestellt, um einen Job zu erledigen. Wenn du es nicht tust, müssen wir deine Anstellung überdenken.
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Wander_High_IG_001_GetYourAss=Komm sofort zurück, sonst bist du gefeuert.
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Wander_High_IG_002_YouFlyAway=Flieg weg von mir und wir sind fertig. Hörst du mich?
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Wander_High_IG_003_HeyIHired=Hey, ich habe dich eingestellt, um mich zu beschützen. Jetzt komm zurück und beschütze mich!
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Wander_Low_IG_001_GetBackOver=Komm hierher zurück, okay? Ich werde es nicht nochmal sagen.
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Wander_Low_IG_002_StopWanderingOff=Hör auf, dich zu verlaufen, wir haben einen Job zu erledigen.
+PU_GENPILOT1_CV_Escort_WarningMissionFail_Wander_Low_IG_003_LookIfYou=Schaue mal, wenn du nicht sofort zurückkommst, überdenke ich ernsthaft deinen Anstellungsstatus.
 PU_GENPILOT1_CV_Escort_WereAllHere_IG_001_EveryoneHereCool=Alle hier? Cool.
-PU_GENPILOT1_CV_Escort_WereAllHere_IG_002_AndThatsEverybody=Und das sind alle.
+PU_GENPILOT1_CV_Escort_WereAllHere_IG_002_AndThatsEverybody=Und das ist jeder.
 PU_GENPILOT1_CV_Escort_WereAllHere_IG_003_WereAllHere=Wir sind alle hier.
-PU_GENPILOT1_CV_Escort_WereAllHere_IG_004_ThatsTheLast=Das ist das Letzte.
-PU_GENPILOT1_CV_Escort_WereAllHere_IG_005_ThatsItEveryone=Das ist es. Alle sind hier.
-PU_GENPILOT1_CV_Escort_WorkFinished_IG_001_ThereWeShould=Da. Wir sollten jetzt gut sein.
-PU_GENPILOT1_CV_Escort_WorkFinished_IG_002_WereAllSet=für dieses Ziel sind wir bereit.
-PU_GENPILOT1_CV_Escort_WorkFinished_IG_003_AndComplete=Uaaaaund... abgeschlossen.
-PU_GENPILOT1_CV_Escort_WorkFinished_IG_004_Done=Fertig.
-PU_GENPILOT1_CV_Escort_WorkFinished_IG_005_ThatllDoIt=Das reicht.
-PU_GENPILOT1_CV_Escort_WorkFinished_UnderAttack_IG_001_DoneGo=Fertig! Gehen!
-PU_GENPILOT1_CV_Escort_WorkFinished_UnderAttack_IG_002_AllSet=Alles klar!
+PU_GENPILOT1_CV_Escort_WereAllHere_IG_004_ThatsTheLast=Das ist der Letzte.
+PU_GENPILOT1_CV_Escort_WereAllHere_IG_005_ThatsItEveryone=Das ist es. Jeder ist hier.
+PU_GENPILOT1_CV_Escort_WorkFinished_IG_001_ThereWeShould=Da. Das sollte reichen.
+PU_GENPILOT1_CV_Escort_WorkFinished_IG_002_WereAllSet=Wurden eingestellt auf dieser Seite.
```

Comment:
Kann original nicht nachschauen grad, hört sich aber komisch an

Reply:
And that's everybody.
Und das sind alle.